verification/curator-service/ui/src/components/bulk-case-form-fields/CaseValidationError.test.ts

```diff
@@ -0,0 +1,29 @@
+import CaseValidationError from './CaseValidationError';
```

Comment:
Perhaps we should also have an integration test so that if the real error messages change we know that we need to change our parsing logic. Though I'm not sure we can trigger backend error messages right now with the UI validation we have in place...

Reply:
Great suggestion. I can add this to the Cypress test edited in a subsequent PR, or, we should be able to check this in the case API supertest tests.